pontoon/teams/views.py

```diff
@@ -129,19 +129,26 @@ def ajax_permissions(request, locale):
             }
         )
 
-    managers = l.managers_group.user_set.all()
-    translators = l.translators_group.user_set.exclude(pk__in=managers).all()
-    all_users = User.objects.exclude(pk__in=managers).exclude(pk__in=translators).exclude(email='')
+    managers = l.managers_group.user_set.order_by('email')
+    translators = l.translators_group.user_set.exclude(pk__in=managers).order_by('email')
+    all_users = (
+        User.objects
+            .exclude(pk__in=managers | translators)
+            .exclude(email='')
+            .order_by('email')
+    )
 
-    contributors_qs = User.translators.with_translation_counts(None, Q(locale=l), None)
-    contributors = set(contributor.email for contributor in contributors_qs)
+    contributors_emails = set(
```

Comment:
@mathjazz I restored set, removed temporary *_qs variable and renamed contributors variable into (hopefully) more descriptive label.

Reply:
@mathjazz, @jotes if we are not using the `translation_counts` and we are not using the ordering supplied in `translation_counts` should we be using this (expensive) method at all?

Im wondering if we should add a manager method specifically to get the emails etc

Reply:
Yeah, we should totally optimize that. Not as part of this PR, though:
https://bugzilla.mozilla.org/show_bug.cgi?id=1379149

Note that I just recently added `with_translation_counts` here, because the original (and more straightforward) version was even slower: https://github.com/mozilla/pontoon/commit/5a7f766a3cc39fd436539db56ce04396ff3de2df.